packages/kg-default-nodes/lib/nodes/header/HeaderNode.js

```diff
@@ -0,0 +1,269 @@
+import {KoenigDecoratorNode} from '../../KoenigDecoratorNode';
+import {renderHeaderNodeToDOM} from './HeaderRenderer';
+import {HeaderParser} from './HeaderParser';
+import {createCommand} from 'lexical';
+
+export const INSERT_HEADER_COMMAND = createCommand();
+
+export class HeaderNode extends KoenigDecoratorNode {
+    // header payload properties
+
+    __size;
+    __style;
+    __buttonEnabled;
+    __buttonUrl;
+    __buttonText;
+    __header;
+    __subheader;
+    __hasHeader;
+    __hasSubheader;
```

Comment:
I don't see these stored in the markdown version, we can probably remove these and just check if the header and subheader are empty instead